test_data/yast/raid1_lvm/lvm+raid1_opensuse.yaml

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
I thought we agreed to use exactly same path structure for schedule and test data. Don't you mind to use same directories (either `yast/raid1_lvm/` or `yast/raid` for both)?

Reply:
I suggested him to use the same for both, but not mix it with existing raid, it is a different test suite anyway, so a folder in both named /lvm+raid1/ should be clear.